### PR TITLE
Enable .NET Framework F5 & VS TestExplorer support for libraries

### PR DESCRIPTION
**Pull request by ViktorHofer**
Tried it out locally, this works now :)

cc @Anipik

### CODE REVIEW

**Review by safern — APPROVED** [review]

File: eng/build.ps1 [hunk 1]
```diff
@@ -85,6 +85,7 @@ if ($vs) {
 
     # This tells .NET Core to use the same dotnet.exe that build scripts use
     $env:DOTNET_ROOT="$PSScriptRoot\..\artifacts\bin\testhost\netcoreapp5.0-Windows_NT-$configuration-$archTestHost";
+    $env:DEVPATH="$PSScriptRoot\..\artifacts\bin\testhost\net472-Windows_NT-$configuration-$archTestHost";
```
Comment (safern) [inline eng/build.ps1]: Is DEVPATH just a thing of Full Framework? If not should we condition it out for netcoreapp and other tfms?

Comment (ViktorHofer) [reply eng/build.ps1]: Yeah that's .NET Framework only.

**Comment by ViktorHofer** [comment]
I just added the DEVPATH environment variable to the launchSettings.json as well (we had that setting in Arcade before but at some point I removed the .NET Framework profile as it was never working). This means that .NET Framework F5 now works without the -vs switch :)

**Comment by ViktorHofer** [comment]
Given that both changes aren't exercised in CI, I will merge this :)

**Pull request merged**